### PR TITLE
UI updates (ready for review)

### PR DESCRIPTION
**Pull request by ackernaut**
Addresses **2nd pass items** for #1833

### CODE REVIEW

**Comment by ackernaut** [comment]
@mattkrick @dan-f this should be a safe to go into today’s release — any volunteers to review? this mostly updates styles, makes certain components more consistent, and cleans up the code a little.

**Comment by dan-f** [comment]
I'll take the review!

**Comment by ackernaut** [comment]
gracias @dan-f !

**Comment by dan-f** [comment]
Ran through the new UI bits, which look good to me! On to taking a look at the code.

File: src/universal/styles/theme/globalStyles.js [hunk 1]
```diff
@@ -47,6 +47,8 @@ export default {
     '-webkit-font-smoothing': 'antialiased'
   },
 
+  ...placeholder,
```
[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/styles/theme/globalStyles.js]: Yes, good point, safe in this case. I wanted to keep it closer to the `input` declaration for context but it really doesn’t help much, moving it back down.

File: src/universal/modules/notifications/components/TaskInvolves.js [hunk 2]
```diff
@@ -112,7 +105,12 @@ class TaskInvolves extends Component {
             </span>
             <span>{':'}</span>
           </div>
-          <div className={taskStyles}>
+          <div className={css(styles.taskListView)}>
```
[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/modules/notifications/components/TaskInvolves.js]: Yeah, I didn’t want to rework the shared styles object that was imported. Punting for now.

File: src/universal/components/Tag/TagPro.js [hunk 3]
```diff
@@ -1,45 +1,41 @@
...
+  textShadow: '0 .0625rem 0 rgba(255, 255, 255, .5)'
+});
 
 const TagPro = (props) => {
-  const {label, styles} = props;
+  const {label} = props;
   return (
-    <div className={css(styles.tagBase)}>
-      <div className={css(styles.tagInner)}>
+    <TagProRoot>
```
[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/components/Tag/TagPro.js]: 👍

File: src/universal/components/IconButton/IconButton.js [hunk 4]
```diff
@@ -24,53 +38,8 @@ IconButton.propTypes = {
   iconName: PropTypes.string,
   iconSize: PropTypes.string,
   onClick: PropTypes.func,
-  styles: PropTypes.object,
   title: PropTypes.string,
   type: PropTypes.string
 };
```
[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/components/IconButton/IconButton.js]: +1

File: src/universal/components/IconButton/IconButton.js [hunk 5]
```diff
@@ -1,21 +1,35 @@
...
-import withStyles from 'universal/styles/withStyles';
-import {css} from 'aphrodite-local-styles/no-important';
 import FontAwesome from 'react-fontawesome';
 import appTheme from 'universal/styles/theme/appTheme';
+import styled from 'react-emotion';
+
+const {warm} = appTheme.palette;
+const color = appTheme.palette.warm40l;
+
+const ButtonBase = styled('button')(({disabled}) => ({
```
[the remark this answers is not in the document]
Comment (ackernaut) [reply src/universal/components/IconButton/IconButton.js]: I’ll give it a try!

**Pull request merged**